src/core/model_config_utils.cc

```diff
@@ -444,11 +444,12 @@ GetBooleanSequenceControlProperties(
         seen_control = true;
 
         if (c.int32_false_true_size() > 0) {
```

Comment:
Have a single test to make sure that exactly one of int, float, or bool is specified.

Reply:
Could you elaborate more on that? So far the if -else statements should be able to test if only one type is specified and return error message if not.

Reply:
Before the code was structured around their being only two choices. If A then must not be B. You've changed it to: if A then must not be B or C, else if B then must not be C, else must be C. Seems better to say. If not (A || B || C) then error. If (A&&B)||(A&&C)||(B&&C) then error.

Reply:
I see. Thank you for the explanation, will modify.